tutorials/pyroot/pyroot002_TTreeAsMatrix.py

```diff
@@ -34,15 +33,15 @@ def make_example():
         tree.Fill()
     root_file.Write()
 
-    return root_file, tree
-
 
 # The conversion of the TTree to a numpy array is implemented with multi-
 # thread support.
 ROOT.ROOT.EnableImplicitMT()
 
 # Create a ROOT file with a tree and the branches "x" and "y"
-_, tree = make_example()
+root_file = ROOT.TFile("pyroot002_example.root", "RECREATE")
+make_example(root_file)
+tree = root_file.Get("tree")
```

Comment:
So what makes the difference is just creating the file outside of `make_example`? Or is it getting the tree outside? Or both?